### PR TITLE
Update the_guide.html

### PR DESCRIPTION
**Pull request by abelgrave**
Updated 'Meeting Timeline' under squad facilitators section @makinde

### CODE REVIEW

File: pages/the_guide.html [hunk 1]
```diff
@@ -184,7 +184,57 @@
...
+                <p><strong>Confidentiality Reminder​.</strong> Everything that's said in the room, stays in the room (see&nbsp;<a href="#squads-rules">language protocol</a>)</p>
+            </td>
+        </tr>
+        <tr>
+            <td><em>10 min</em></td>
+            <td><strong>Give members 10 minutes to write out their updates.</strong> Each person should write out 8 bits of information, personally and professionally. Feel free to play background music to set the tone.</td>
+        </tr>
+        <tr>
+            <td><em>30 min</em></td>
+            <td>E<strong>ach person takes a turn sharing their updates​ (2-3 min each).</strong> Remind members that they have max 3 min and that as they share, let you know if they would like to add any challenges to the presentation queue.</td>
```
Comment (astonm) [inline pages/the_guide.html]: Lonely `E` here

File: pages/the_guide.html [hunk 2]
```diff
@@ -184,7 +184,57 @@
...
+    <table class="table">
+    <thead>
+        <tr><th width="100px">Timing</th><th>Summary</th></tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><em>Arrive</em></td>
+            <td>
+                <p><strong>Arrive Early.</strong> Get there at least 15 minutes in advance. Make sure the space is setup. Arrange the chairs in a circle and verify that everything is good to go.</p>
+                <p><span id="docs-internal-guid-a555ef64-a252-0417-8ba3-87fae66e1c17"><strong>Start on Time.</strong><span> It is important that members time is respected. You should start right on time, and end right on time. This will require monitoring the time throughout the meeting.</span></span></p>
```
Comment (makinde) [inline pages/the_guide.html]: What's this `id` attribute? I'm guessing you can delete it.

Comment (abelgrave) [inline pages/the_guide.html]: id has been removed.

File: pages/the_guide.html (hunk 1 above)
Comment (abelgrave) [inline pages/the_guide.html]: Updated.

**Pull request merged**